src/webview/js/generatedEs3.js

```diff
@@ -142,7 +145,8 @@ var sendScrollMessage = function sendScrollMessage() {
 };
 
 var sendScrollMessageIfListShort = function sendScrollMessageIfListShort() {
-  if (documentBody.scrollHeight === documentBody.clientHeight) {
+  var messageListHeight = document.getElementById('message-list').clientHeight;
+  if (messageListHeight <= documentBody.clientHeight) {
```

Comment:
This doesn't seem right -- take a look at the documentation of `clientHeight` and `scrollHeight`, on MDN. We should be doing a comparison on the total height of the message list, aka its `scrollHeight`.

In particular I believe this condition will always be true. With a strict inequality (`<`), it would be true when any of the other children in the body, like the loading spinners, are on screen.